packages/teams-js/src/public/app.ts

```diff
@@ -806,6 +816,98 @@ export namespace app {
       resolve(sendAndHandleStatusAndReason('executeDeepLink', deepLink));
     });
   }
+
+  /**
+   * A namespace for enabling the suspension or delayed termination of an app when the user navigates away.
+   * When an app registers for the registerBeforeSuspendOrTerminateHandler, it chooses to delay termination.
+   * When an app registers for both registerBeforeSuspendOrTerminateHandler and registerOnResumeHandler, it chooses the suspension of the app .
+   * Please note that selecting suspension doesn't guarantee prevention of background termination.
+   * The outcome is influenced by factors such as available memory and the number of cached apps.
+   *
+   * @beta
+   */
+  export namespace lifecycle {
+    /**
+     * Register on resume handler function type
+     *
+     * @param context - Data structure to be used to pass the context to the app.
+     */
+    type registerOnResumeHandlerFunctionType = (context: ResumeContext) => void;
+
+    /**
+     * Register before suspendOrTerminate handler function type
```

Comment:
>Register before suspendOrTerminate handler function type [](http://example.com/codeflow?start=7&length=56)

Should the comment here describe the expectation on this function? E.g., "When this function is called, your app should do any cleanup needed before suspension or termination. Upon returning, the app will be suspended or terminated." ?

Reply:
Added the comment :)